lib/astupdate.js

```diff
@@ -1,122 +1,139 @@
 /*
   Functions in this file are used to update the AST to support for pattern matching
 */
-const estemplate = require('./estemplate')
-const updateComment = require('./updateComment.js')
+const estemplate = require("./estemplate");
+const updateComment = require("./updateComment.js");
 
-let patterns = []
+let patterns = [];
 
 const buildExplicitCases = (funcdecl, patterndecls) => {
-  let patterncases = []
-  let flag = false
+  let patterncases = [];
+  let flag = false;
   for (let pattern of patterndecls) {
-    let patterncase = {'type': 'SwitchCase', 'consequent': []}
-    let patternParam = pattern.declarations[0].init.params[0]
-    patterncase.test = patternParam
-    if (patternParam.type === 'ArrayExpression') {
-      patterncase.test = estemplate.literal('0')
+    let patterncase = { type: "SwitchCase", consequent: [] };
+    let patternParam = pattern.declarations[0].init.params[0];
+    patterncase.test = patternParam;
+    if (patternParam.type === "ArrayExpression") {
+      patterncase.test = estemplate.literal("0");
       patterncase.test.sType = {
-        'type': 'array',
-        'elemTypes': {},
-        'commonType': 'needsInference',
-        'isHomogeneous': false
-      }
-      flag = true
+        type: "array",
```

Comment:
Can this be modified to follow StandardJS? We prefer using single quotes and no semicolons (except where absolutely required)

Reply:
Changed to single quotes